app/assets/stylesheets/common/base/menu-panel.scss

```diff
@@ -702,6 +704,23 @@ body.footer-nav-ipad {
 }
 
 .menu-panel.slide-in {
+  top: var(--header-top);
+  box-sizing: border-box;
+
+  /* Use dvh where supported, with fallback to vh */
+  --100dvh: 100vh;
+  --100dvh: 100dvh;
```

Comment:
Very nice! We should probably lift this to `variables.scss` and use more liberally.

Reply:
All our officially targeted browsers support `dvh`, so I'd prefer not to solidify the fallback forever in `variables.scss`. I added it here for chrome < v108 (pre-november-2022) since this is such a critical part of the app. Given that Chrome is 'evergreen', I hope we can drop it in a couple of months once people have had more time to update their browsers. Safari and Firefox have supported it for much longer.